puppet/modules/jenkins_job_builder/files/centos.org/ansible/ssh_config.j2

```diff
@@ -0,0 +1,10 @@
+{% for host, value in cico_data.results.hosts.items() %}
```

Comment:
Could we put this file in a 'templates' directory?

Reply:
Will do